### ISSUE TITLE
Source is not published

### ISSUE THREAD

**Issue by whiskey1** [PinIdea/ASIC-X11-Miner#2]
Under release, the source isn't there. those source.zip or tar.gz files have all of the executables in them, but no source. same with git pulls.  

I want to see the source to see if I can't move the driver bits over for a mac compile.

**Comment by soleo** [comment]
a possible workaround solution would be using ubuntu releases to build docker images, and use the images to launch the mining software on Mac OS

**Comment by whiskey1** [comment]
Yeah, i'm on the docker OSX beta with zero USB support. So its running in Parallels - Ubuntu. I'm getting about half the listed speed. I'll bring this into my datacenter later this week and plug into actual linux servers for a real test.

running 1.0.7 now, bit more stable too. Also, might want to add that users with multiple DU1 devices can just add  "-S /dev/ttyUSB0 -S /dev/ttyUSB1" etc... on one command line to get them to all operate as one.

**Comment by soleo** [comment]
Great. Could you make a Pull Request for your dockerfile so that everyone else would be benefit from it?
